_includes/configuration/formats.md

```diff
@@ -132,7 +132,12 @@ For each format some additional parameters can be specified:
 
 #### `classes`
 
-Space-separated list of classes that are applied to the selected or newly created inline/block element.
+Space-separated list of classes that are:
+
+- Applied to newly created inline or block elements.
+- Appended to selected existing inline or block elements.
```

Comment:
Appended to selected existing inline or block elements.

maybe change of order?

Appended to existing selected inline or block elements.

Reply:
> Appended to existing selected inline or block elements.

This reads oddly to me

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1831/commits/739e20d7cf784a9fb7fad6acca85d3c12d482cf9